frontend/src/pages/Landing/Footer.tsx

```diff
@@ -0,0 +1,70 @@
+import React, { ReactElement } from 'react';
+import styles from './landing.module.css';
+import { dti_logo } from '../../icons/other';
+
```

Comment:
I think there should be an explicit import for the images used in the footers here (e.g. `import group525 from './Group525.png'`) since the images were not showing up for me otherwise and it makes the code cleaner by avoiding hard-coding paths in the body of the code.